main.tex

```diff
@@ -858,7 +865,7 @@ \subsubsection*{Provenance activities and relations}
 Ideally, it also takes an explicit requirement into account, which, in some cases, if formally defined, can also be used for calibrating the simulation model~\cite{Palaniappan2013, Mitra2019}.
 It may also use an \A{}.
 
-In the case of the activity~\texttt{CSM1} of \lee{}~\cite{Lee2003}, several wet-lab data from their own experiments (\texttt{WD1}--\texttt{WD5}) as well as from Salic et al. (2000)~\cite{Salic2000} (\texttt{WD1}--\texttt{WD3}) are being used during the calibration of the model \texttt{SM1} which produces a \SE{}~\texttt{SE1}, the corresponding \SD{}~\texttt{SD1} as well as the calibrated \SM{}~\texttt{SM2}.
+In the case of the activity~\texttt{CSM1} from \lee{}~\cite{Lee2003}, several wet-lab data from their own experiments (\texttt{WD1}--\texttt{WD5}) as well as from Salic et al. (2000)~\cite{Salic2000} (\texttt{WD1}--\texttt{WD3}) are used during the calibration of the model \texttt{SM1} which produces a \SE{}~\texttt{SE1}, the corresponding \SD{}~\texttt{SD1} as well as the calibrated \SM{}~\texttt{SM2}.
```

Comment:
I've changed "of" to "from" in several situations where I thought it was a better word choice